runners/flink/flink_runner.gradle

```diff
@@ -422,3 +422,8 @@ createPipelineOptionsTableTask('Python')
 // Update the pipeline options documentation before running the tests
 test.dependsOn(generatePipelineOptionsTableJava)
 test.dependsOn(generatePipelineOptionsTablePython)
+
+// delegate spotlessApply to :runners:flink:spotlessApply
+tasks.named("spotlessApply") {
+  dependsOn ":runners:flink:spotlessApply"
```

Comment:
Will this have the right source sets? I would expect the sources for each subdirectory to need its own task with the overridden source set. (and yes i see this will conflict with the top-level source set - not sure what is the best solution)

Reply:
Yes, this is somewhat due to how flink runner is built. Spotless is applied using `:runners:flink:spotlessApply`, although the runner is built using e.g. `:runners:flink:1.18:build`. This only makes the `:runners:flink:spotlessApply` called automatically for all versions.